config/locales/en.yml

```diff
@@ -49,6 +49,12 @@ en:
       will_buy_diapers: "diapers to be used in the future"
       money_spent: "UAH already spent on diapers"
       money_will_be_spent: "UAH yet to be spent on diapers"
+      used_diapers: "одноразових підгузків уже використано"
+      will_be_used_diapers: "одноразових підгузків ще буде використано"
+      pieces: "pcs"
+      used_money: "витрачено на одноразові підгузки станом на сьогодні"
+      money_will_be_used: "ще буде витрачено до моменту відмови від одноразових підгузків"
+      hryvnia: "UAH"
```

Comment:
why here isn't English translations?

Reply:
Currently, there is only the Ukrainian version, English doesn`t provided yet.